plugins/oauth/components/LoginPage.js

```diff
@@ -122,7 +122,8 @@ class LoginPage extends mixin(StoreMixin) {
                 target="_blank"
               >
                 documentation
-              </a>.
+              </a>
+              .
```

Comment:
@brandonc , are we not losing translations here?

Reply:
I re-ran the extraction and there were no changes.

Reply:
Okay, cool.